src/components/new-members/index.js

```diff
@@ -16,19 +17,18 @@ const renderNewUserCard = (newMember) => {
   );
 };
 
-const renderNewUser = (newMember, isSuperUser) => {
+const renderNewUser = (newMember, isSuperUser, handleNewMemberDetailsPage) => {
```

Comment:
can we name it handleNewMemberClick?

Reply:
yes we can name it, will name it and push the changes again.